app/views/items/index.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1>My Items</h1>
+<% @items.each do |item| %>
+  <p> <%= item.name %> </p>
```

Comment:
doing like this were just going through all items that exist and also lose data for our merchant so we cant call @merchant to get the info we need to match the wire frame